content/docs/user-guide/testing/best-practices/_index.md

```diff
@@ -155,33 +164,40 @@ int CalculateTestSquare( const int number)
 
 TEST(MathTests, FastSquare_Integer_Squared1)
 {
+    // Assumes the function is correct, thus only proves an identity property
     int tenSquared = Math::FastSquare(10);
     int result = Math::FastSquare(10);
     ASSERT_EQ(result, tenSquared);
 }
 
 TEST(MathTests, FastSquare_Integer_Squared2)
 {
+    // Relies on a duplicate implementation of the production code that contains
+    // the same bug, thus only proves the production bug was not changed
     int tenSquared = CalculateTestSquare(10);
     int result = Math::FastSquare(10);
     ASSERT_EQ(result, tenSquared);
 }
 
 TEST(MathTests, FastSquare_Integer_Squared3)
 {
+    // Uses incorrectly computed squares from another file, thus only
+    // proves the production bug was not changed
     int result = Math::FastSquare(10);
     ASSERT_EQ(result, pcs_ten);
 }
 
 TEST(MathTests, FastSquare_Integer_Squared4)
 {
+    // directly states that 10^2 = 20, which is incorrect but also *easy to notice*
```

Comment:
Since this is in a code block, italics don't render 
```suggestion
    // directly states that 10^2 = 20, which is incorrect but also easy to notice
```

Reply:
Had added the asterisks for emphasis. Keeping unless you disagree with them rendering as asterisks.